src/App.js

```diff
@@ -26,21 +27,31 @@ class App extends Component {
 
   componentDidMount() {
     this.sessionManager.initialize(this.updateSession);
+    this.setUrlLanguage(this.state.dopplerSession);
```

Comment:
Parameter is not necessary, right?

```suggestion
    this.setUrlLanguage();
```

Reply:
Thanks! Fixed